fee_allocator/accounting/recon.py

```diff
@@ -10,77 +10,69 @@
 from fee_allocator.accounting import PROJECT_ROOT
 
 
-def recon_and_validate(
-    fees: dict,
-    fees_to_distribute: dict,
-    timestamp_now: int,
-    timestamp_2_weeks_ago: int,
-    aura_vebal_share: Optional[Decimal] = Decimal(0.506),
-) -> None:
+def recon_and_validate(fees: dict, fees_to_distribute: dict, timestamp_now: int, timestamp_2_weeks_ago: int,
+                       target_aura_vebal_share: Decimal) -> None:
     """
     Recon fees collected from the fee pipeline. Store the summary to json file
     and raise exceptions if validation fails
     """
     # Move to separate function
     all_fees_sum = Decimal(round(sum(fees_to_distribute.values()), 4))
-    all_incentives_sum = round(sum(
-        [
-            sum(
-                [
-                    x["fees_to_vebal"],
-                    x["fees_to_dao"],
-                    x["aura_incentives"],
-                    x["bal_incentives"],
-                ]
-            )
-            for x in fees.values()
-        ]
-    ), 4)
+    all_incentives_sum = round(
+        sum([sum([x["fees_to_vebal"], x["fees_to_dao"], x["aura_incentives"], x["bal_incentives"], ]) for x in
+             fees.values()]), 4)
     # If everything is 0 - don't store the summary
     if all_incentives_sum == 0 or all_fees_sum == 0:
         return
-    aura_incentives = sum([x["aura_incentives"] for x in fees.values()])
-    bal_incentives = sum([x["bal_incentives"] for x in fees.values()])
-    fees_to_dao = sum([x["fees_to_dao"] for x in fees.values()])
-    fees_to_vebal = sum([x["fees_to_vebal"] for x in fees.values()])
     delta = all_fees_sum - all_incentives_sum
-
     abs_delta = abs(delta)
```

Comment:
Broke down all the loops above into a single loop that has room to continue checking more below.  Added checks for negative values, as this I got caught by it this time and once before.  The deltas can add up if there are large positrives and then negatives, but you can't pay a negative amount of incentive.